round4/team-05.md

```diff
@@ -0,0 +1,162 @@
+# Chrome Extension Idea: [FlyHistory]
+
+## Authors
+
+Melanie Chen\
+Noah Erdogan\
+Jason Figueroa\
+Brady Zhou
+
+## Problem Statement
+
+Finding flights is often very time consuming, and users often find themselves opening many different tabs. FlyHistory provides an easy way to keep track of your flight search, so that you can easily revisit what you looked for in the past.
+
+## Target Audience
+
+- People looking for flights
+
+## Description
+
+FlyHistory will save the history of flights users have looked through on google flights. From there users will have the ability to filter their flight view history based on location, price, luggage type and other flight information. They can then revisit these flights saved in their history Users will also have the ability to remove flights from their history that they are no longer interested in.
+
+## Selling Points
+
+1. Gives a single place to look for all your previously searched flights
+2. Simple and intuitive interface for users
+3. Customizable filters for each user
+4. Displays results in a clean and understandable fashion
+
+
+## User Stories
+
+1. As a user, I want a way to automatically save information about flights when searching the web, so that I can easily revisit them in the future.
+
+2. As a user, I want to be able to delete specific flights in my history, so that I can hide them or clean up my interface.
+
+3. As a user, I want to be able to filter by price, so that I can easily find all the flights within my price range.
+
+4. As a user, I want to store the relevant information about my flight such as price, airports, time, etc., so that I can get an overview of a flight.
+
+5. As a user, I would like to add additional settings to avoid adding specific flights to my history, so that my history only involves what I am interested in.
+
+6. As a frequent traveler, I want to filter my flight history by destination, so that I can quickly find past searches for trips to my favorite locations.
+
+7. As someone with specific luggage needs, I want to filter my flight history based on luggage allowances, so that I can find the most accommodating airlines without re-entering search criteria.
+
+8. As a traveler with flexible dates, I want the option to sort my flight history by date ranges, so that I can compare prices across different periods and find the cheapest options.
+
+9. As a user who values efficiency, I want the ability to quickly access detailed information on any flight in my history, so that I can make informed decisions without re-searching the flight.
+
+10. As a user who often compares flights, I want the ability to compare selected flights from my history side-by-side, so that I can easily see differences in price, timing, and other relevant details
+ 
+11. As a user with a preference for certain seats, I want to see information on seat availability and type (window, aisle, extra legroom) for flights in my history, so I can make bookings based on my seating preferences.
```

Comment:
It would be helpful to outline how exactly you plan to get this information (seat availability, and later you mentioned meal options) that is not present in the google search. Do you plan to get the information elsewhere or through an API?